docs/ReverseSSHtunnel.md

```diff
@@ -52,7 +66,7 @@ sudo nano /etc/ssh/sshd_config
 Make sure these entries are active (meaning there is no `#` at the beggining of the line).
 Alternatively, you can just paste these on the end of the file:
 
-```
+```text
```

Comment:
text ?

Reply:
There's no other highlighting format for it, but we can go ahead and remove it – the result will be the same.

Reply:
As you wish.
Just didn't understand the purpose, that's all.

Reply:
Removed it :)